exasol_script_languages_container_ci/lib/release.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import os
+from pathlib import Path
+
+import click
+from exasol_integration_test_docker_environment.lib.base import luigi_log_config
+from exasol_integration_test_docker_environment.lib.config import build_config
+
+from exasol_script_languages_container_ci.lib.ci_build import ci_build
+from exasol_script_languages_container_ci.lib.ci_push import ci_push
+from exasol_script_languages_container_ci.lib.ci_test import ci_test
+from exasol_script_languages_container_ci.lib.github_release_asset_uploader import GithubReleaseAssetUploader
+from exasol_script_languages_container_ci.lib.release_upload import release_upload
+
+
+def release(ctx: click.Context,
+            flavor: str,
+            docker_user: str,
+            docker_password: str,
+            docker_release_repository: str,
+            config_file: str,
+            source_repo_url: str,
+            release_id: int,
+            release_uploader: GithubReleaseAssetUploader,
+            is_dry_run: bool):
+    """
+    Run Release build:
+    1. Build image
+    2. Run basic tests
+    3. Push to docker release repository
+    4. Upload to GH release url
+    """
+    logging.info(f"Running Release build for parameters: {locals()}")
+
+    flavor_path = (f"flavors/{flavor}",)
+
+    log_path = Path(build_config.DEFAULT_OUTPUT_DIRECTORY) / "jobs" / "logs" / "main.log"
+    os.environ[luigi_log_config.LOG_ENV_VARIABLE_NAME] = f"{log_path.absolute()}"
+
+    ci_build(ctx, flavor_path=flavor_path, rebuild=True, build_docker_repository="",
+             commit_sha="", docker_user="", docker_password="")
+    ci_test(ctx, flavor_path=flavor_path)
+    if not is_dry_run:
```

Comment:
In try run log that the push should happen here

Reply:
Done